### ISSUE TITLE
DOH and Logs is gone

### ISSUE THREAD

**Issue by phanluchoaofficial1152** [ragibkl/adblock-dns-server#212]
Hello,

I have try to access ur logs link of all servers and i got error about ssl is failed and cannot access to it, doh too. Can u give a look with this ?

Thanks

**Comment by ragibkl** [comment]
Hi @phanluchoaofficial1152 , 

I think logs are working fine. You can ignore SSL errors.
From the faq page https://bancuh.com/faq, let me use the following example for sg-dns1:
```
sg-dns1
DoT: sg-dns1.bancuh.com
DoH: https://sg-dns1.bancuh.com/dns-query
Do53-ipv4: 139.59.219.245
Do53-ipv6: 2400:6180:0:d0::f7:1001
Logs-ipv4: http://139.59.219.245:8080/logs
Logs-ipv6: http://[2400:6180:0:d0::f7:1001]:8080/logs
Logs-ssl: http://sg-dns1.bancuh.com:8443/logs
```

If you click any of the following:
- Logs-ipv4: http://139.59.219.245:8080/logs
- Logs-ipv6: [http://[2400:6180:0:d0::f7:1001]:8080/logs](http://[2400:6180:0:d0::f7:1001]:8080/logs)

They will bring you to the logs page for sg-dns1, using ipv4 or ipv6 addresses.
SSL does not work for web access via ip address.
There's no way to add that support.
Use these links if you want to force the logs page to detect your ipv4 or ipv6 public ip address.
I added this because some people have both ipv4/ipv6, and maybe they want to view logs for each one separately.
Your browser google chrome or firefox will complain, so you have to click something to tell it to open the page anyway.
Something like browse anyway or ignore error.

If you click the link: http://sg-dns1.bancuh.com:8443/logs, SSL should work. This uses actual ssl certificate.
However, you have no control which ipv4 ipv6 the server detect on your public ip address.

I hope this explains the situation for you.

If it still does not work, please share the screenshot from your screen, and let's work it out.
Also let me know which link you are using.

Otherwise, please close this ticket.

**Comment by phanluchoaofficial1152** [comment]
Yeah, it works right now but ur blacklist domains do not update manually ?
Can u use blacklist domain list updated daily on website dnsforfamily.com ?
They update blacklist domain daily in 24 hours. Thanks.

Vào Th 2, 14 thg 10, 2024 lúc 08:45 Ragib Badaruddin <
***@***.***> đã viết:

**Comment by ragibkl** [comment]
That list looks pretty big, about 6 million. I'm worried that my servers won't be able to handle that big list.

@Tomatoide , what do you think?

Also @phanluchoaofficial1152 , why don't you just use their DNS server for yourself? You don't always have to use my servers. Other people also provide pretty good DNS service!

Or, maybe you can try their DNS for a few days, and report back here how it feels?

**Comment by phanluchoaofficial1152** [comment]
Yeah, i have tried their dns but the thing that i dont use it because their
server located on location that very far with me and it takes me make more
time for request its server to my isp --> my internet got slow after use
it. I use ur server because is near of my location so i got the best
results. Their server i try to ping google.com is takes 230ms to solve the
website - i dont like it. My network is 250mbps but i have to wait for a
long time to access web or app when using their dns. If ur server can,
please add it. If it cannot, it ok i will try to use their dns way by way.

Vào Th 2, 14 thg 10, 2024 lúc 17:23 Ragib Badaruddin <
***@***.***> đã viết:
